### PR TITLE
Modified file command-line-interface

### PR DESCRIPTION
**Pull request by rudri182**
Modified file as per the comments in #643. The issue for this PR is #727.

### CODE REVIEW

File: command-line/command-line-interface.md [hunk 1]
```diff
@@ -134,13 +134,13 @@ We tend to use kebab-case for directory (folder) names and git repository names.
 8. Use `TAB key` to autocomplete the names of directories and files while in the command line faster as it'll autocomplete the string as far as it can before you have to disambiguate. 
     Hit the tab twice to see the list of AUTOCOMPLETE POSSIBILITES (if multiple matches).
```
Comment (alodahl) [inline command-line/command-line-interface.md]: Could you change this all caps phrase to *italic* or  **bold**?

Comment (alodahl) [reply command-line/command-line-interface.md]: Same with the absolute path and relative path headers below... I’d suggest a 

### header 

or 

#### header

**Comment by alodahl** [comment]
Some insight into using all caps... for UIs (front end user interfaces) this is a big no-no. This is because if someone were to be using a screen reader to read it, it would spell it out for them, which is obnoxious. Hearing “U.I.” works, but hearing “A.B.S.O.L.U.T.E.”  would be awful.  If all caps is part of the style of the page, the solution is to use css to text-transform it into all caps, which still lets screen readers use it normally. 🤓

**Comment by rudri182** [comment]
I have made changes according to the comments @alodahl. Please review it. 😊

File: command-line/command-line-interface.md [hunk 2]
```diff
@@ -134,13 +134,13 @@ We tend to use kebab-case for directory (folder) names and git repository names.
...
    `open` is not magically connected to Mac's Finder, but actually it's just using whatever default program on your Mac opens a file with that extension, and in the case of directories, the default program is Finder.
 
-11. Another mac shortcut `cmd+ shift+ g` can be used anywhere you see a Finder window (even in other Mac programs such as SourceTree or VSCode when you're opening a file or directory). This brings up a little text box into which you can type or paste a path. Use `pwd` to get your current path, then copy it from the Terminal and open up sourcetree, go to file > open, then use `cmd + shift + g` to paste in the path you want.    
+11. Another Mac shortcut `cmd+ shift+ g` can be used anywhere you see a Finder window (even in other Mac programs such as SourceTree or VSCode when you're opening a file or directory). This brings up a little text box into which you can type or paste a path. Use `pwd` to get your current path, then copy it from the terminal and open up SourceTree, go to file > open, then use `cmd + shift + g` to paste in the path you want.    
 
 12. `ls` command is used to peek into other folders.
        The list of ls argumnets are
```
[the remark this answers is not in the document]
Comment (Kundan28) [reply command-line/command-line-interface.md]: I missed that... 😞

**Pull request merged**